code/modules/uplink/uplink_items.dm

```diff
@@ -1359,6 +1359,13 @@ datum/uplink_item/stealthy_weapons/taeclowndo_shoes
 	cost = 2
 	exclude_modes = list(/datum/game_mode/nuclear, /datum/game_mode/nuclear/clown_ops)
 
+/datum/uplink_item/suits/roller_skates
+	name = "Roller Skates"
+	desc = "Shoes with hidden roller blades to speed away from the crime. Note - Roller blades are not hidden"
+	item = /obj/item/clothing/shoes/rollers
+	cost = 3
+	exclude_modes = list(/datum/game_mode/nuclear) //Nukies dont get cool things
```

Comment:
What the fuck trillby I want 5 terrorists on rollerskates and I want them now.

Reply:
They already get so much in terms of speed, and making them even faster just dosnt seem all that good. Also 45% not 50% speed buff. If its to brokenly good ill make it 20%